doc/source/_includes/rllib/new_api_stack.rst

```diff
@@ -1,7 +1,7 @@
 .. note::
 
-    Ray 2.10.0 introduces the alpha stage of RLlib's "new API stack".
-    The team is currently transitioning algorithms, example scripts, and
-    documentation to the new code base throughout the subsequent minor releases leading up to Ray 3.0.
+    Ray 2.40 uses :doc:`RLlib's new API stack </rllib/rllib-new-api-stack>` by default.
+    The team has mostly completed transitioning algorithms, example scripts, and
+    documentation to the new code base.
 
-    :doc:`See here </rllib/rllib-new-api-stack>` for more details on how to activate and use the new API stack.
+    In case you are still using the old API stack, :doc:`see here </rllib/new-api-stack-migration-guide>` for more details on how to migrate.
```

Comment:
I still think we should give a certain date - maybe even just a month in 2025 in which we do not support anymore the old stack.

Reply:
We need to see. Can't do that so easily b/c of a certain customer.